cql/ManageRareAbnormality.cql

```diff
@@ -1627,7 +1627,7 @@ define RecommendationForHistologyResults:
       )
     ) then
       {
-        short: 'See Details',
+        short: 'Surveillance',
         date: Today(),
         group: 'Managing Histology (I.5)',
         details: {
```

Comment:
Here the short is 'Surveillance' but details says 'observation' - no mention of HPV testing or cotesting, which is how Surveillance is used elsewhere. Should details be changed to 'surveillance with primary HPV testing or cotesting' ?